src/main/java/com/revature/studyforce/flashcard/dto/FlashcardDTO.java

```diff
@@ -0,0 +1,54 @@
+package com.revature.studyforce.flashcard.dto;
+
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.flashcard.model.Topic;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.util.Assert;
+import java.sql.Timestamp;
+import java.util.function.Function;
+
+/**
+ * Flashcard Data Transfer Object
+ *
+ * @author Luke
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlashcardDTO {
+    private int flashcardId;
+    private User creator;
+    private Topic topic;
+    private String question;
+    private int questionDifficultyTotal;
+    private int questionDifficultyAverage;
+    private Timestamp createdTime;
+    private Timestamp resolutionTime;
+    private boolean isResolved;
+
+    /**
+     * Converts Flashcard to FlashcardDTO
+     * @return - FlashcardDTO
+     */
+    public static Function<Flashcard, FlashcardDTO> convertToDTO() {
```

Comment:
incorrect documentation, this method does not convert a flashcard to DTO nor does it return a flashcardDTO. This method acts as a getter which returns a function which can then be used to convert a flashcard to its DTO counterpart